server/src/main/java/com/linecorp/centraldogma/server/internal/api/MirroringServiceV1.java

```diff
@@ -125,11 +133,21 @@ private CompletableFuture<PushResultDto> createOrUpdate(String projectName,
         });
     }
 
+    // Mirroring may be a long-running task, so we need to increase the timeout.
+    @RequestTimeout(value = 5, unit = TimeUnit.MINUTES)
+    @Post("/projects/{projectName}/mirrors/{mirrorId}/run")
+    public CompletableFuture<MirrorResult> runMirror(@Param String projectName, @Param String mirrorId)
```

Comment:
Question) Is there no need to at least limit the people who can run mirrors to those who own the project?

```suggestion
    @RequiresWritePermission
    public CompletableFuture<MirrorResult> runMirror(@Param String projectName, @Param String mirrorId)
```

Reply:
We need it.